src/css/override_components.scss

```diff
@@ -135,15 +135,20 @@ button,
   }
 } /* stylelint-enable */
 
+.usa-button-disabled {
```

Comment:
Hmm, `action-cautious` isn't correct either. This might be a larger specificity issue we can't solve now. We can keep the `usa-disabled` class in.